modules/ocf_stats/templates/munin/mail-munin-alert.epp

```diff
@@ -107,3 +112,9 @@ def main(argv):
 
 if __name__ == '__main__':
     main(argv)
+
+
+# Local Variables:
+# mode: python
+# End:
+# vim: ft=python
```

Comment:
Is all of this needed? Isn't there a one-line addition that's usually just used at the bottom of files instead of all of this?

Reply:
The first three are for emacs, the last one for vim